src/Sylius/Component/Core/Model/ChannelInterface.php

```diff
@@ -89,4 +89,11 @@ public function addCountry(CountryInterface $country): void;
     public function removeCountry(CountryInterface $country): void;
 
     public function hasCountry(CountryInterface $country): bool;
+
+    /**
+     * @return Collection|CountryInterface[]
+     *
+     * @psalm-return Collection<array-key, CountryInterface>
```

Comment:
Since psalm was dropped from the project
```suggestion
     * @return Collection<array-key, CountryInterface>
```